messenger_users/views.py

```diff
@@ -79,6 +80,35 @@ def confirm_linkedin_profile(request):
 # def browse_saved_jobs(request):
 #     pass
 
+def search_linkedIn_profiles(request):
+    data = json.loads(request.body)
+    messenger_id = data['messenger_id']
+    username = data['linkedin_username']
```

Comment:
This shouldn't take into input of someone's linkedin_username, but rather take into someon's first_name and last_name. We are trying to make this as easy as possible for them